airflow/providers/celery/executors/celery_executor.py

```diff
@@ -143,6 +142,7 @@ def __getattr__(name):
 )
 
 # worker cli args
+ARG_AUTOSCALE = Arg(("-a", "--autoscale"), help="Minimum and Maximum number of worker to autoscale")
```

Comment:
Seems unrelated to the PR no?

Reply:
yes, somewhat, but it was defined with the `# roles` args, and is actually only used here, so I moved it to its logical home.